### DISCUSSION TITLE
Possible furniture for sims stuff

### DISCUSSION THREAD

**Discussion by jwright159** [Dragon-Fox-Collective/SBEPIS#177] [General]
- Bed
- Dresser
- Table
- Endtable / nightstand
- Chest
- Dining chair
- Mirror
- Pillow
- Couch
- TV
- Bookshelf
- Sink
- Bathtub
- Shower
- Toilet
- Lava lamp
- Wheel of cheese
- Fridge
- Curtains
- Posters
- Plants
- Oven
- Books/photos/knicknacks/clutter for shelves
- Piano
- Plushies
- Computer
- Desk
- Lamp

**Comment by DietermiGamzD125** [comment]
((I thoroughly applaud the lava lamp idea))
